### ISSUE TITLE
Bulk retry crashes from web

### ISSUE THREAD

**Issue by jc00ke** [sorentwo/oban#377]
### Environment

- oban 2.3.2
- oban_pro 0.5.1
- oban_web 2.3.0

- PostgreSQL Version: 11.9
- Elixir & Erlang/OTP Versions (`elixir --version`): 1.11.2 w/OTP 23

### Current Behavior

After I select multiple discarded jobs and retry them, the stats go wonky and I'm pretty sure the LV crashes. Takes a refresh to continue.

![image](https://user-images.githubusercontent.com/18191/99608943-c70fe000-29c3-11eb-9817-c5e6874e5170.png)

I happened to notice this while I was the console:

```
00:49:59.209 [error] GenServer {Oban.Registry, {Oban, {:plugin, Oban.Pro.Plugins.BatchManager}}} terminating
** (ArgumentError) argument error
    :erlang.function_exported({:ok, BasicIncome.Workers.Transfer}, :handle_discarded, 1)
    (oban_pro 0.5.1) lib/oban/pro/plugins/batch_manager.ex:143: Oban.Pro.Plugins.BatchManager.insert_callback/3
    (oban_pro 0.5.1) lib/oban/pro/plugins/batch_manager.ex:84: Oban.Pro.Plugins.BatchManager.handle_info/2
    (stdlib 3.13.2) gen_server.erl:680: :gen_server.try_dispatch/4
    (stdlib 3.13.2) gen_server.erl:756: :gen_server.handle_msg/6
    (stdlib 3.13.2) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Last message: {:debounced, "202011"}
```

### Indexes

```

"oban_jobs_pkey" PRIMARY KEY, btree (id)
"oban_jobs_args_vector" gin (to_tsvector('english'::regconfig, args::text))
"oban_jobs_attempted_at_id_index" btree (attempted_at DESC, id) WHERE state <> ALL (ARRAY['available'::oban_job_state, 'scheduled'::oban_job_state, 'executing'::oban_job_state, 'retryable'::oban_job_state])
"oban_jobs_queue_state_priority_scheduled_at_id_index" btree (queue, state, priority, scheduled_at, id)
"oban_jobs_worker_gist" gist (worker gist_trgm_ops)
```

**Comment by sorentwo** [comment]
This was a bug in Oban Pro. Closed with Oban Pro v0.5.2 👍

**Comment by jc00ke** [comment]
![](https://media1.tenor.com/images/0845ca0aba488a791031adc7f0d702c8/tenor.gif)
